src/main/java/com/ridanisaurus/eecreateaddon/EECreateDataGen.java

```diff
@@ -247,13 +247,13 @@ protected void buildTags(Consumer<IFinishedGenericJSON> consumer) {
 			for (MaterialModel material : registry.getMaterials()) {
 				List<String> processedType = material.getProcessedTypes();
 				// Crushed Ores
-				if (processedType.contains("crushed_ore")) {
+				if (processedType.contains("crushed_raw")) {
 					ResourceLocation crushedOre = EECreateRegistrar.crushedOreMap.get(material.getId()).getId();
-					if (!createCrushedOres.contains("#create:crushed_ores/" + material.getId())) createCrushedOres.add("#create:crushed_ores/" + material.getId());
```

Comment:
The issue in this PR is that this is the one of few lines that are for the actual minecraft tags, everything else is EE type that is in the configuration file, what when changed, will break compatibility with configs from the old version and cause modpack developers to change all materials with `crushed_ore` proccessing type and break scripts that are working on old recipe id >.>

Reply:
I forgot this existed.
Regarding your comment: I know that but that is because of the breaking change from create side.
When this was implemented I had an issue with a personal pack that after upgrading create no ore recipe worked with create.